grafana-plugin/src/containers/UserSettings/parts/index.tsx

```diff
@@ -133,7 +133,6 @@ export const TabsContent = observer(({ id, activeTab, onTabChange, isDesktopOrLa
         ) : (
           <PhoneVerification userPk={id} />
         ))}
-      {/* TODO: we should probably hide this tab when a user (ie. Admin) is viewing the user settings for another user. Would it make sense for an Admin to be able to link their mobile app to another user's profile */}
```

Comment:
this comment was outdated. This case is already being handled [here](https://github.com/grafana/oncall/blob/dev/grafana-plugin/src/containers/UserSettings/UserSettings.tsx#L56)